src/providers/cambridge/get_test_data.py

```diff
@@ -0,0 +1,39 @@
+import os
```

Comment:
what's the difference between this and src/providers/cambridge/cambridge_cli.py? Would you like to just test it?

Reply:
Yes, just for testing.